onecodex/models/generated.py

```diff
@@ -0,0 +1,1527 @@
+
+
+class DocumentSchema(BaseModel):
+    field_uri: str = Field(
+        ...,
+        alias="$uri",
+        examples=["/api/v1/documents/0d77065796f8d173"],
+        pattern="^/api/v1/documents/[a-f0-9]{16}$",
+        title="$Uri",
+    )
+    created_at: str = Field(
+        ...,
+        description="Timestamp for when the object was created on the One Codex platform, encoded as a [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) timestamp.",
+        title="Created At",
+    )
+    description: Optional[str] = Field(None, description="ExcludeFromOpenAPI", title="Description")
+    downloaders: List[Downloader] = Field(
+        ...,
+        description="An (optionally empty) array of references to users that the document has been shared with. These users are able to download (but not modify) the document.",
+        title="Downloaders",
+    )
+    filename: str = Field(
+        ..., description='The document filename (e.g., "report.pdf")', title="Filename"
+    )
+    size: Optional[Size] = Field(
+        ..., description="The size of the document in bytes.", title="Size"
+    )
+    uploader: Uploader = Field(
+        ...,
+        description='A reference to the user that uploaded and owns the document, e.g., `{"$ref": "/api/v1/users/5891ee65711c4d5e"}`. Only owners can modify a document (with some exceptions in the case of organization accounts configured for multiple users - please [contact us](mailto:support@onecodex.com) if you\'d like to discuss this use case).',
+        title="users",
+    )
+    valid_until: Optional[str] = Field(None, description="ExcludeFromOpenAPI", title="Valid Until")
+
+
+class Status(Enum):
+    success = "success"
+    failed = "failed"
+    pending = "pending"
+    retryable = "retryable"
+
+
+class EventSchema(BaseModel):
+    field_uri: str = Field(
+        ...,
+        alias="$uri",
+        examples=["/api/v1/events/0d77065796f8d173"],
+        pattern="^/api/v1/events/[a-f0-9]{16}$",
+        title="$Uri",
+    )
+    payload: Dict[str, Any] = Field(..., description="The webhook payload.", title="Payload")
+    status: Status = Field(
+        ..., description="The status of the webhook for the event.", title="Status"
+    )
+
+
+class FunctionalRunSchema(BaseModel):
+    field_uri: str = Field(
+        ...,
+        alias="$uri",
+        examples=["/api/v1/functional_profiles/0d77065796f8d173"],
+        pattern="^/api/v1/functional_profiles/[a-f0-9]{16}$",
+        title="$Uri",
+    )
+    complete: Optional[bool] = Field(False, title="Complete")
+    # cost: Optional[CostSchema] = Field(..., description="ExcludeFromOpenAPI")
+    created_at: str = Field(
+        ...,
+        description="Timestamp for when the object was created on the One Codex platform, encoded as a [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) timestamp.",
+        title="Created At",
+    )
+    # dependencies: List[Dependency] = Field(
```

Comment:
Why some fields are commented out?

Reply:
Yeah. Hand rolling does not sound as fancy and "we are very clever in this team" as auto-generating does. But then we will need an another step: code transformation to filter out generated code.

Complexity upon complexity and hand rolling starts sounding way better.

Reply:
> Yeah. Hand rolling does not sound as fancy and "we are very clever in this team" as auto-generating does. But then we will need an another step: code transformation to filter out generated code.

> Complexity upon complexity and hand rolling starts sounding way better.

I _think_ I agree, but let's discuss at/right after standup and then I'll move those changes into this PR _or_ not based on the discussion.